src/material-experimental/mdc-slider/slider.ts

```diff
@@ -385,7 +397,11 @@ export class MatSlider implements AfterViewInit, OnChanges, OnDestroy, ControlVa
 
   ngOnDestroy() {
     this._dirChangeSubscription.unsubscribe();
-    this._foundation.destroy();
+    // The foundation cannot be destroyed on the server, as the foundation
+    // has not be initialized on the server.
+    if (this._platform.isBrowser) {
+      this._foundation.destroy();
```

Comment:
Maybe just check `if (this._foundation)`? That way if we add another condition to the initialization logic it won't break.

Reply:
The foundation is always created, so the check would always be true. If we would only create the foundation on the browser, I guess we'd need to make it "null"-able and handle it in more places.